src/modules/forms/plugins/RenderPlugins.js

```diff
@@ -121,6 +122,12 @@ class RenderPlugin extends Component {
                       {...opts}
                     />
                   )
+                case 'cb':
```

Comment:
I think we can move this switch statement to something like this. Wdyt?

```javascript
const PluginComponents =  {
    'cb': CircuitBreaker,
    'fp': FancyPlugin,
      ...
}

const PluginComponent = ({ name, options }) => {
    const component = PluginsComponents[name];
    return component ? <component {...options} /> : null;
};

// Usage:
<PluginComponent name={pluginName} options={opts} />
```

Reply:
Greate idea, @ngenerio !!!
But here should be upperCase 😉 
```js
return Component ? <Component {...options} /> : null;
```